services/movie_session.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       cinema_hall_id=cinema_hall_id,
+                                       movie_id=movie_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    movies_sessions_queryset = MovieSession.objects.all()
+    if session_date:
+        movies_sessions_queryset = movies_sessions_queryset.filter(
+            show_time__date=session_date
+        )
+    return movies_sessions_queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
you can rewrite this logic for efficiency, so you don't have to get session and save it in case if you didn't get any optional params. Also returning movie_session is not a requirement